python/tvm/auto_scheduler/measure.py

```diff
@@ -624,12 +647,7 @@ def local_build_worker(args):
         The build result of this Builder thread.
     """
     inp, build_func, timeout, verbose = args
-    if build_func == "default":
-        build_func = tar.tar
-    elif build_func == "ndk":
-        build_func = ndk.create_shared
-    else:
-        raise ValueError("Invalid build_func" + build_func)
```

Comment:
we could add one check `assert any(build_func == name for name in  BuildFunc.nam)`